docs/algorithms/GenericLineConflation.asciidoc

```diff
@@ -276,7 +276,8 @@ are merged.
 
 [[GenericConflationQuality]]
 .Conflation Quality
-image::algorithms/GenericLineTestGraph.png[Random Forest vs. Generic Line Road Conflation Performance,scalewidth="50%"]
+image::algorithms/images/GenericLineTestGraph.png[Random Forest vs. Generic Line Road Conflation Performance,scalewidth="50%"]
+////
```

Comment:
The commented out `gnuplot` code actually shows up in the document.  The `////` is the opening and closing of a multi-line comment.  There is an old ticket to replace it but that hasn't happened so I don't want to delete it, just keep it from being visible in the PDFs.